core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -124,15 +108,16 @@ export class TdsTableBodyRow {
       <Host
         class={{
           'tds-table__row': true,
+          'selected': this.selected,
```

Comment:
Just a reminder to remove of prefix this class, since its on host and outside of shadow dom.